provision-horde.sh

```diff
@@ -0,0 +1,490 @@
+#!/bin/sh
+
+# shellcheck disable=1091
+. mail-toaster.sh || exit
+
+export JAIL_START_EXTRA=""
+# shellcheck disable=2016
+export JAIL_CONF_EXTRA=""
+export JAIL_CONF_EXTRA="
+        mount += \"$ZFS_DATA_MNT/horde \$path/data nullfs rw 0 0\";
+        mount += \"$ZFS_DATA_MNT/vpopmail \$path/usr/local/vpopmail nullfs rw 0 0\";"
+
+mt6-include 'php'
+mt6-include nginx
+
+install_horde()
+{
+
+    if [ ! -d "$ZFS_DATA_MNT/horde/data" ]; then
+        tell_status "creating $ZFS_DATA_MNT/horde/data"
+        mkdir "$ZFS_DATA_MNT/horde/data"
+    fi
+
+    tell_status "making vpopmail dir"
+    mkdir -p "$STAGE_MNT/usr/local/vpopmail"  || exit
+
+    install_php 56 || exit
+	install_nginx || exit
+    
+	tell_status "installing Horde IMP and Ingo "
+	stage_pkg_install horde-ingo
+    stage_pkg_install horde-imp
+    stage_pkg_install php56-simplexml php56-ftp php56-gd php56-fileinfo php56-tidy
```

Comment:
You can add your required PHP modules to the `install_php 56` line above like so:

`install_php 56 "simplexml ftp gd fileinfo tidy"` The install_php function will take care of expanding the modules to the port names with the php version.